data/recommendation/notify.js

```diff
@@ -1,68 +1,140 @@
+const yo = require('yo-yo');
+
 class Notify {
   start() {
-    this.setup();
+    this.target = this.createBase();
+    document.body.appendChild(this.target);
+    const methodsToBind = ['handleEndWarning', 'handleEndSignup'];
+    for (let key of methodsToBind) { // eslint-disable-line prefer-const
+      this[key] = this[key].bind(this);
+    }
     self.port.on('data', recs => {
-      this.showAdvice(recs);
+      const date = new Date(recs.BreachDate);
+      const year = date.getFullYear();
+      this.render({
+        warn: true,
+        site: recs.Title,
+        year,
+      });
     });
+    this.hasSync = false;
     self.port.on('syncEnabled', () => {
-      this.removeSyncSignup();
+      this.hasSync = true;
     });
     self.port.on('syncDisabled', () => {
-      this.showSyncSignup();
+      this.hasSync = false;
```

Comment:
These should trigger a re-render, in case this values come in after the initial render. Maybe something like

```js
this.updateState({ hasSync: false });
```

which would update an instance variable, and then trigger a re-render?